### PR TITLE
fix(select, checkbox): error & success style (VIV-1458)

### PR DESCRIPTION
**Pull request by rachelbt**
what was done?
1. styled select ghost error + success as it is in Figma:
image
2. added style for checkbox error + success - need to be approved by designers
image

### CODE REVIEW

File: libs/components/src/lib/select/select.scss
Comment (RichardHelm) [inline libs/components/src/lib/select/select.scss:1]: Can we add a screenshot for this?

**Review by RichardHelm — CHANGES_REQUESTED** [review]
Please see comment

File: libs/components/src/lib/select/select.scss
Comment (rachelbt) [reply libs/components/src/lib/select/select.scss:1]: you mean for the ghost select states?
I guess we need to.

**Review by RichardHelm — APPROVED** [review]

**Comment by rachelbt** [comment]
@RichardHelm  - changed  the code for checkbox according to Ayala's request of design

**Review by RichardHelm — APPROVED** [review]

**Pull request merged**